### PR TITLE
DIABLO-63, sign-up page takes shape

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-63

Next, the form and the necessary API.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -25,12 +25,39 @@
...
+          </tr>
+        </tbody>
+      </template>
     </v-data-table>
   </div>
 </template>
 
 <script>
   import Utils from '@/mixins/Utils'
+  import router from "@/router"
```
Comment (paulkerschen) [inline src/views/Home.vue]: "Double quotes!"

File: src/views/Home.vue [hunk 2]
```diff
@@ -52,23 +81,27 @@
       this.courses = []
       this.coursesEligibleOnly = []
       this.$_.each(this.$currentUser.teachingSections, s => {
+        const isEligible = s.isEligibleForCourseCapture
         let course = {
           course: `${s.courseName}, ${s.instructionFormat} ${s.sectionNum}`,
-          title: s.courseTitle,
-          schedule: this.describeSchedule(s),
+          days: s.meetingDays ? this.$_.join(s.meetingDays, ", ") : undefined,
```
Comment (paulkerschen) [inline src/views/Home.vue]: "Ditto"

File: src/views/SignUp.vue [hunk 3]
```diff
@@ -1,23 +1,80 @@
...
+              <v-col>
+                {{ oxfordJoin($_.map(section.instructors, 'name')) }}
+              </v-col>
+            </v-row>
+            <v-row v-if="section.meetingDays">
+              <v-col md="auto">
+                <v-icon>mdi-calendar</v-icon>
+              </v-col>
+              <v-col>
+                {{ $_.join(section.meetingDays, ", ") }}
```
Comment (paulkerschen) [inline src/views/SignUp.vue]: "as above"

**Comment by paulkerschen** [comment]
Thanks, @johncrossman, punctuation noted.

**Comment by johncrossman** [comment]
Thank you @paulkerschen  Amended. I also added 'quotes' rule to `.eslintrc.js`

**Comment by paulkerschen** [comment]
download

**Pull request merged**